src/test/mochitest/head.js

```diff
@@ -617,10 +617,17 @@ function togglePauseOnExceptions(
  * @return {Promise}
  * @static
  */
-function invokeInTab(fnc) {
+function invokeInTab(fnc, args) {
   info(`Invoking function ${fnc} in tab`);
-  return ContentTask.spawn(gBrowser.selectedBrowser, fnc, function*(fnc) {
-    content.wrappedJSObject[fnc](); // eslint-disable-line mozilla/no-cpows-in-tests, max-len
+  return ContentTask.spawn(gBrowser.selectedBrowser, fnc, function*(fnc, args) {
+    content.wrappedJSObject.apply(func, args); // eslint-disable-line mozilla/no-cpows-in-tests, max-len
```

Comment:
Should this be `fnc`, not `func`?

Reply:
OK, so there are two problems here:

1.  It should be `fnc`, not `func`
2.  `TypeError: content.wrappedJSObject.apply is not a function` bricks any test that uses `invokeInTab`.  So, for example, if I dumb it way down with the following, all tests run:

```js
function invokeInTab(fnc, args) {
  info(`Invoking function ${fnc} in tab`);
  return ContentTask.spawn(gBrowser.selectedBrowser, fnc, function*(fnc, args) {
    // content.wrappedJSObject.apply(fnc, args); // eslint-disable-line mozilla/no-cpows-in-tests, max-len
    console.log('invoking in tab: ', fnc, args);

    // FIX HERE
    if(args) {
      content.wrappedJSObject[fnc](...args);
    }
    else {
      content.wrappedJSObject[fnc]();
    }
  });
}
```

Reply:
A call like `ContentTask.spawn(browser, arg, task)` only passes `task` a single argument, which is a structured clone of `arg`. So I think we could do something like this:
```
function invokeInTab(fnc, ...args) {
  info(`Invoking in tab: ${fnc}(${args.map(uneval).join(',')})`);
  return ContentTask.spawn(gBrowser.selectedBrowser, { fnc, args }, function*({ fnc, args }) {
    content.wrappedJSObject[fnc](...args); // eslint-disable-line mozilla/no-cpows-in-tests, max-len
  });
}

```